scripts/ci/upload_artifacts.sh

```diff
@@ -10,16 +10,16 @@ then
   if [[ "${CI_TAG}" ]];
   then
     if [[ ${ALL_FILES_ACCESS} == "ON" ]]; then
-      #FILENAME_AAB="qfield-${CI_TAG}-${TRIPLET}.aab"
-      FILENAME_APK="qfield-${CI_TAG}-${TRIPLET}.apk"
+      #FILENAME_AAB="qfield-all-access-${CI_TAG}-${TRIPLET}.aab"
+      FILENAME_APK="qfield-all-access-${CI_TAG}-${TRIPLET}.apk"
```

Comment:
this part is already handled in the other script, no?
you should be able to remove the conditions and just use `APP_PACKAGE_NAME`

Reply:
basically:

```
  if [[ "${CI_TAG}" ]];
  then
    #FILENAME_AAB="${APP_PACKAGE_NAME}-${CI_TAG}-${TRIPLET}.aab"
    FILENAME_APK="${APP_PACKAGE_NAME}-${CI_TAG}-${TRIPLET}.apk"
  else
    #FILENAME_AAB="${APP_PACKAGE_NAME}-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${TRIPLET}.aab"
    FILENAME_APK="${APP_PACKAGE_NAME}-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${TRIPLET}.apk"
  fi
```